### PR TITLE
Noref fix deletions

### PR DESCRIPTION
**Pull request by nonword**
Fix bug where indexer failed to delete anything due to an apparent bug in latest `elasticsearch` library (probably a support gap for our very old ES version).

Also fixes tiny checkin card format parsing bug.

### CODE REVIEW

File: lib/utils/suppressBibs.js [hunk 1]
```diff
@@ -6,16 +6,15 @@ const suppressBib = async (bib) => {
...
+  logger.debug(`Deleting: ${bibUri}`)
 
-  return elasticClient.delete({
+  const opts = {
     index: process.env.ELASTIC_RESOURCES_INDEX_NAME,
     type: 'resource',
     id: bibUri
-  })
+  }
+  return elasticClient.delete(opts)
```
Comment (charmingduchess) [inline lib/utils/suppressBibs.js:17]: is this not an async call? do we want to await it?

File: lib/elastic-search/requests.js [hunk 2]
```diff
@@ -107,10 +107,10 @@ const currentDocument = async (bibId, index) => {
   const record = await client.search({
     index,
     body: {
-      query: { term: { uris: bibId } }
+      query: { term: { uri: bibId } }
```
Comment (charmingduchess) [inline lib/elastic-search/requests.js:110]: was this the bug?

Comment (nonword) [reply lib/elastic-search/requests.js:110]: We no longer index `uris` (containing both bnum and all item nums), bug we've always indexed `uri` (just the bnum), so anywhere we're querying `uris` needs to change to `uri`

File: lib/utils/suppressBibs.js (hunk 1 above)
Comment (nonword) [reply lib/utils/suppressBibs.js:17]: It is an async call, but we don't need to do anything with the resolved value in this function, so it's fine to return the unresolved Promise

**Review by charmingduchess — APPROVED** [review]

**Pull request merged**